website_logo/controllers/main.py

```diff
@@ -80,5 +80,5 @@ def website_logo(self, dbname=None, **kw):
                             image, filename=imgname, mtime=mtime)
                         return response
             except Exception:
-                return http.send_file(placeholder(imgname))
+                pass
```

Comment:
Hmmm guess I need to re-read that one, but I agree this was bad. `.error` it is, there is indeed nothing to do because the end return handles it